upup/pkg/fi/cloudup/scalewaytasks/instance.go

```diff
@@ -177,6 +179,66 @@ func (_ *Instance) RenderScw(c *fi.CloudupContext, actual, expected, changes *In
 		if err != nil {
 			return fmt.Errorf("error waiting for instance %s of group %q: %w", srv.Server.ID, fi.ValueOf(expected.Name), err)
 		}
+
```

Comment:
It may be a bit confusing but `newInstanceCount` is calculated at line 126, it's either :

- `> 0`, then we need to add `newInstanceCount` instances (line 130), or 
- `< 0`, then we need to remove `- newInstanceCount` instances (line 245)